### ISSUE TITLE
File loader output is generated into the wrong folder

### ISSUE THREAD

**Issue by queicherius** [stefanprobst/next-svg#1]
It looks like the output when falling back to the file loader is generated into `.next/server/static/images`, but Next expects it in `.next/static/images`.

```
    "next": "^11.0.0",
    "@stefanprobst/next-svg": "^1.1.3",
```

**Comment by stefanprobst** [comment]
thanks for reporting!

looks like this changed with next.js 11 - i'll try to update soon (currently peer dependencies are still set to `"next": "^9.5.0||^10.0.0"`).

**Comment by stefanprobst** [comment]
ok, this is not next.js 11 but webpack 5 - in case it's urgent this package should still work with next.js 11 with `{ webpack5: false }` in `next.config.js`.

**Comment by queicherius** [comment]
Awesome, thanks!

Edit: Just an FYI, if you are testing it, this is currently required in the `next.config.js`:

```js
  // FIX: https://github.com/vercel/next.js/issues/25950
  images: { disableStaticImages: true },
```

**Comment by stefanprobst** [comment]
thanks!

could you try if it works for you with `@stefanprobst/next-svg@next`?

**Comment by queicherius** [comment]
Yep, that one works perfectly. 💯

**Comment by stefanprobst** [comment]
great! now published as version 2.0.0.
